.changeset/great-parents-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/localizations': patch
+---
+
+Review PT-BR localization
```

Comment:
Can we both update the PR title + changeset to mention "Fix typos and add missing keys to pt-BR localization"?

"Review" is a bit generic for the changes introduced here.